### DISCUSSION TITLE
Atom Cam V2 / Wyze Cam V3 Family Reunion

### DISCUSSION THREAD

**Discussion by virmaior** [gtxaspec/wz_mini_hacks#120] [General]
I haven't set it up yet but I did open the box.

Here are some pictures:
![DSC_0727](https://user-images.githubusercontent.com/70625876/174997555-34030521-dcde-48fe-ad2a-ff20463f8235.JPG)
![DSC_0729](https://user-images.githubusercontent.com/70625876/174997586-51bd31f4-1ad8-4d96-bce1-bc543e57462e.JPG)
![DSC_0731](https://user-images.githubusercontent.com/70625876/174997592-38e18f51-d997-4f1b-a77b-f5da47608d27.JPG)
![DSC_0732](https://user-images.githubusercontent.com/70625876/174997597-52330e24-c480-427d-a884-cff4d1807b8b.JPG)
![DSC_0733](https://user-images.githubusercontent.com/70625876/174997608-df605461-7d54-43d1-bae6-3208ce31ebc6.JPG)
![DSC_0734](https://user-images.githubusercontent.com/70625876/174997615-986a4cee-9a81-4012-81f2-9fdc4e6f6b8c.JPG)
![DSC_0735](https://user-images.githubusercontent.com/70625876/174997619-488b8190-c2e6-45a1-a83b-54b1d53924f3.JPG)
![DSC_0736](https://user-images.githubusercontent.com/70625876/174997628-235bac52-6db6-4410-a6d9-9b3d56ed652f.JPG)
![DSC_0740](https://user-images.githubusercontent.com/70625876/174997635-e7c5bcd9-0a16-45fb-a8a2-deea98a2acdc.JPG)

It's *very* similar but also minutely different.

Visual impressions:
1. bottom footer thing is slightly larger on Atom V2
2. wyze v3 has a permanent tail as for its microusb / Atom v2 has a port
3. the SD card slot is a lot easier to open on the Atom V2 / it is also a lot harder to close
4. the white border around the black part is way smaller on the atom v2

Both have the use our app to do first setup arrangement (atom app is available for phones on us google play).
Also the mac address starts in a similar range if memory serves (7c ...)

Hopefully I can get some time to boot it up (will also be sending one to our gtxaspec per his request!)

**Comment by virmaior** [comment]
Have now set it up:

(1) First screen: enter email address in box, click button.
(2) Click red text 同意する
(3) Click on 認定
(4) Enter number from email
(5) pick a password between 8 and 32 characters(
6) click red button デバイス追加
(7) click bottom entry "Atom Cam 2"
(8) remove film (prompt) (click 次へ at bottom)
(9) click checkbox, push reset until you hear you'll hear setsuzoku machi desu and then 次 at bottom
(10) click 許可
(11) enter wifi info (2.4ghz only -- I'm not saying I forgot this and had to do it a second time ...)
(12) click 確認
(13) It will say QR code wo ninshiki dekimasita shibaraku wo machi kudasai (QRコードを認識できました。しばらくを待ちください。)
(14) Check button at bottom and click 次へ
(15) it will enter a screen that has a countdown from 120 seconds
(16) It will give a message that means it was able to confirm connecting to wifi (wifiにつながることできました。後にご確認ください

and then it's good to go.

The process is basically the same as wyze enough so that it's obviously the same code base for setup. Similar enough that I wonder what would happen if I just tried to use wyze's app to set it up.

**Reply by endertable** [reply]
It's funny how many of these cams are made by different vendors with their own little finishing touches.  That would be interesting trying to use Wyze's app.  The only thing I could see would be that wyze's app is going to try to use their servers to try to find the camera, but the camera probably will use atom's server to provision itself?  In effect, they would both get stuck and not acknowledge each other, but this kind of stuff is interesting nonetheless.

**Reply by virmaior** [reply]
yeah, I almost went ahead and tried that, but I didn't want to find out if that would work and end up with a brick.

**Reply by mrlt8** [reply]
The biggest problem with the Atom/Neos cams is that the apps are region locked to different app stores, so you'll need to switch regions/app stores to download the app.

Not sure if things have changed, but I actually picked up an Atom v2 some time ago for testing with my bridge, and, other than the voice prompt and emails, I believe the whole setup process and app interface were all in English.

**Reply by virmaior** [reply]
I'm in Japan but I have American google play store. App is available on the American store.

Nothing was in English in the setup for Atom v3 (not a problem for me). Since  (annoyingly for me) apps on android default to the menu language which I've got set to English forcing me to suffer the impoverished translations to English of things, I don't think there's an English version of the atom setup app on android.

**Reply by mrlt8** [reply]
You're right. I just reinstalled the app from the US App Store. Must have been the Neos cam that was only available in the UK store. Everything is still in English for me including the setup on iOS.
